README.md

```
@@ -3,7 +3,7 @@
 [![Java](https://img.shields.io/badge/Java-17-%23ED8B00.svg?logo=openjdk&logoColor=white)](#)
 [![Spring Boot](https://img.shields.io/badge/Spring%20Boot-3.2.5-6DB33F?logo=springboot&logoColor=fff)](#)
 [![JUnit5](https://img.shields.io/badge/Tested_with-JUnit_5-1?logo=junit5)](#)
-[![Linear](https://img.shields.io/badge/Task_tracker_we_prefer-Linear-5E6AD2?logo=linear&logoColor=fff)](#)
+[![Quality gate](https://sonarcloud.io/api/project_badges/quality_gate?project=vrnsky_java-a-to-z)](https://sonarcloud.io/summary/new_code?id=vrnsky_java-a-to-z)[![Linear](https://img.shields.io/badge/Task_tracker_we_prefer-Linear-5E6AD2?logo=linear&logoColor=fff)](#)
```

Markdownlint (0.35.0)

No empty links

(MD042, no-empty-links)